instrumentation/rabbitmq-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/RabbitTracer.java

```diff
@@ -66,7 +66,9 @@ public Context startGetSpan(
           SemanticAttributes.MESSAGING_DESTINATION,
           normalizeExchangeName(response.getEnvelope().getExchange()));
       if (CAPTURE_EXPERIMENTAL_SPAN_ATTRIBUTES) {
-        spanBuilder.setAttribute("rabbitmq.routing_key", response.getEnvelope().getRoutingKey());
+        spanBuilder.setAttribute(
+            SemanticAttributes.MESSAGING_RABBITMQ_ROUTING_KEY,
```

Comment:
If this attribute is already a part of the "official" semantic convention, then it is not experimental anymore and should be moved outside this `if`.

Reply:
Done.